### PR TITLE
Add initial version

### PR DESCRIPTION
**Pull request by nelsonmestevao**
I want feedback on this version :pray:

### CODE REVIEW

**Comment by nelsonmestevao** [comment]
I can add the in the next one :+1: I will just try to fix the bottom black bar then :muscle:

**Comment by herulume** [comment]
lgtm
@apedrob since you have more experience, anything you want to add?

**Comment by alves-luis** [comment]
Everything seems fine (besides the favicon 👍)

File: src/styles/_base.scss [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  margin: 0;
+  padding: 5px;
+  text-decoration: none;
+  color: #EAEAEA;
+  transition: 0.5s ease;
+  border-bottom: 1px solid transparent;
+  text-transform: uppercase;
+}
+
+@media (max-width: 620px) {
```
Comment (apedrob) [inline src/styles/_base.scss]: image
We should improve the image/font size and padding in mobile.

File: src/components/SocialLinks.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faFacebook,
+  faTwitter,
+  faInstagram,
+  faGithub,
+} from '@fortawesome/free-brands-svg-icons';
+
+export default function SocialLinks(props) {
```
Comment (apedrob) [inline src/components/SocialLinks.js]: In my opinion, we should maintain the export default in the bottom for future purposes (wrapper, other functions, ..)

Comment (apedrob) [reply src/components/SocialLinks.js]: Normally we do the arrow function like this: `SocialLinks = (props) => { code }`
In this case we are passing all the props down; we often avoid passing all props, for clarity mostly:
`.. SocialLinks = ({facebook, twitter, instagram, github}) => { code }`

File: src/styles/_social.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.social-container {
+  margin: 10px;
+  padding: 25px 25px;
```
Comment (apedrob) [inline src/styles/_social.scss]: My suggestion is putting `padding: 20px`

File: src/styles/_base.scss [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    display: none;
+  } */
+
+  body {
+    /* background-image: url("../img/background-footer.png"); */
+    background-color: #000000;
+  }
+
+  .logo img {
+    max-width: 320px;
```
Comment (apedrob) [inline src/styles/_base.scss]: I think `max-width: 320px;` is enough to solve our problem

File: src/styles/_base.scss [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+  }
+
+  .date {
+    font-size: 24px;
+  }
+
+  .location {
+    font-size: 20px;
+  }
+}
```
Comment (apedrob) [inline src/styles/_base.scss]: I don't know if these sizes are recommended somehow, but we should decrease the values. My suggestion is: `24px`, `20px`, `18px`

File: src/styles/_base.scss (hunk 1 above)
Comment (apedrob) [reply src/styles/_base.scss]: image

File: src/components/SocialLinks.js (hunk 2 above)
Comment (nelsonmestevao) [reply src/components/SocialLinks.js]: Should I do `const SocialLinks = ({facebook, twitter, instagram, github}) => { code }`, then? (_My question is the keyword before SocialLinks_)

Comment (apedrob) [reply src/components/SocialLinks.js]: Yes, creating a _arrow function component_.  The arrow syntax is optional, some people say it's more clean, scope safety and compact. We should define our default syntax (personally I prefer the fat arrow)
But in the end, these components do the same:
`const foo = () => (...);`
`function foo() {
  return ...;
}`

Comment (nelsonmestevao) [reply src/components/SocialLinks.js]: Tell me if it's better now :pray:

**Pull request merged**